tests/release_candidate_scraper/test_release_candidate_scraper_indices.py

```diff
@@ -6,53 +6,22 @@
 
 from mozdownload import ReleaseCandidateScraper
 
-import mozhttpd_base_test as mhttpd
+import pytest
 
 
-test_params = [
-    # -a firefox -p win32 -v 23.0.1 --build-number=1
-    {'args': {'application': 'firefox',
-              'build_number': '1',
-              'platform': 'win32',
-              'version': '23.0.1'},
-     'build_index': 0,
-     'build_number': '1',
-     'builds': ['build1']
-     },
-    # -a firefox -p mac -v 23.0.1 --build-number=3
-    {'args': {'application': 'firefox',
-              'build_number': '3',
-              'platform': 'mac',
-              'version': '23.0.1'},
-     'build_index': 0,
-     'build_number': '3',
-     'builds': ['build3']
-     },
-    # -a firefox -p linux -v 23.0.1 --build-number=2
-    # Invalid build-number
-    {'args': {'application': 'firefox',
-              'build_number': '2',
-              'platform': 'linux',
-              'version': '23.0.1'},
-     # build_index = len(parser.entries) - 1
-     'build_index': 1,
-     'build_number': '2',
-     'builds': ['build1', 'build3']
-     }
-]
+@pytest.mark.parametrize("args,build_index,build_number,builds", [
+    ({'application': 'firefox', 'build_number': '1', 'platform': 'win32', 'version': '23.0.1'},
+     '0', '1', ['build1']),
+    ({'application': 'firefox', 'build_number': '3', 'platform': 'mac', 'version': '23.0.1'},
+     '0', '3', ['build3']),
+    ({'application': 'firefox', 'build_number': '2', 'platform': 'linux', 'version': '23.0.1'},
+     '1', '2', ['build1', 'build3']),
+])
 
+def test_build_indices(setUp, tmpdir, args, build_index, build_number, builds):
+    """Testing indices in choosing builds for ReleaseCandidateScraper"""
 
-class ReleaseCandidateScraperTest_build_indices(mhttpd.MozHttpdBaseTest):
-    """test indices in mozdownload scraper class"""
-
-    def test_build_indices(self):
-        """Testing indices in choosing builds for ReleaseCandidateScraper"""
-
-        for entry in test_params:
-            scraper = ReleaseCandidateScraper(destination=self.temp_dir,
-                                              base_url=self.wdir,
```

Comment:
Note that the base url has to be specified to the server. Do that with the before-mentioned call to `httpd.get_url()`.

Reply:
You didn't address that comment.

Reply:
Please review the changes I've made. I think I've resolved this.

Reply:
Yes, this looks fine.